modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -352,6 +352,8 @@ def _set_columns(self, columns):
 
     def fillna(
```

Comment:
Do we need to add a test for omnisci backend?

Reply:
No, we have separate tests for omnisci backend that do not relate to tests for pandas backend in any way. You should add new tests to [here](https://github.com/modin-project/modin/blob/master/modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py#L486).

Reply:
This PR does changes only to pandas backend and none of them are related to omnisci. I think additions to omnisci tests should be done in some different PR.

Reply:
Makes sense